project/npda/views/patient.py

```diff
@@ -40,43 +40,50 @@ class PatientListView(
     permission_denied_message = "You do not have the appropriate permissions to access this page/feature. Contact your Coordinator for assistance."
     model = Patient
     template_name = "patients.html"
+    paginate_by = 5
```

Comment:
This feels a bit low?

Reply:
yes - sorry left over after testing. I will change to 50